### ISSUE TITLE
Partial visibility doesn't work correctly for < 2 months

### ISSUE THREAD

**Issue by Aeroporas** [airbnb/HorizonCalendar#104]
If I select visibility range in partially visible mode like 

```
let startDate = calendar.date(from: DateComponents(year: 2021, month: 02, day: 5))!
let endDate = calendar.date(from: DateComponents(year: 2021, month: 03, day: 15))!
```

the scrolling function stops working at all and part of the calendar just stucks behind bottom of the screen. The problem is reproducible in example project.

Here is a screenshot

<img width="484" alt="Снимок экрана 2021-02-26 в 17 00 31" src="https://user-images.githubusercontent.com/21102640/109309301-251a5200-7854-11eb-8365-1bf458758ffa.png">

**Comment by Aeroporas** [comment]
When I extend visible dates range all seems to work nice again.

**Comment by bryankeller** [comment]
Oh whoops, definitely a bug. I'll have a fix for this soon. Thanks for the report!

**Comment by bryankeller** [comment]
Thanks for your patience on this @Aeroporas - the fix is in `v1.8.3` 🙂
